tests/fingerprints_tests.py

```diff
@@ -99,73 +105,104 @@ def test_maccs_keys_fingerprint(example_molecules):
     assert np.all(X_maccs == X_seq)
 
 
-def test_atom_pair_fingerprint(example_molecules):
+def test_atom_pair_bit_fingerprint(example_molecules):
     X = example_molecules
+    X_for_rdkit = [Chem.MolFromSmiles(x) for x in X]
+    rdkit_generator = fpgens.GetAtomPairGenerator()
+    atom_pair = AtomPairFingerprint(fingerprint_type="bit", n_jobs=-1)
+    fp_function = rdkit_generator.GetFingerprint
+    X_emf = atom_pair.transform(X.copy())
+    X_rdkit = np.array([fp_function(x) for x in X_for_rdkit])
+    assert np.all(X_emf == X_rdkit)
 
-    X_2 = X.copy()
 
-    # Concurrent
-    atom_pair = AtomPairFingerprint(result_type="default", n_jobs=-1)
-    atom_pair_hashed = AtomPairFingerprint(result_type="hashed", n_jobs=-1)
-    atom_pair_as_bit_vect = AtomPairFingerprint(
-        result_type="as_bit_vect", n_jobs=-1
+def test_atom_pair_sparse_fingerprint(example_molecules):
+    X = example_molecules
+    X_for_rdkit = [Chem.MolFromSmiles(x) for x in X]
+    rdkit_generator = fpgens.GetAtomPairGenerator()
+    atom_pair = AtomPairFingerprint(
+        fingerprint_type="bit", sparse=True, n_jobs=-1
     )
-    X = np.array([Chem.MolFromSmiles(x) for x in X])
-    X_atom_pair = atom_pair.transform(X.copy())
-    X_atom_pair_hashed = atom_pair_hashed.transform(X.copy())
-    X_atom_pair_as_bit_vect = atom_pair_as_bit_vect.transform(X.copy())
+    fp_function = rdkit_generator.GetFingerprint
+    X_emf = atom_pair.transform(X.copy())
+    X_rdkit = csr_array([fp_function(x) for x in X_for_rdkit])
+    assert np.all(X_emf.toarray() == X_rdkit.toarray())
 
-    # Sequential
-    X_2 = [Chem.MolFromSmiles(x) for x in X_2]
-    X_seq = np.array([GetAtomPairFingerprint(x) for x in X_2])
-    X_seq_hashed = np.array([GetHashedAtomPairFingerprint(x) for x in X_2])
-    X_seq_as_bit_vect = np.array(
-        [GetHashedAtomPairFingerprintAsBitVect(x) for x in X_2]
-    )
 
-    assert np.all(X_atom_pair == X_seq)
-    assert np.all(X_atom_pair_hashed == X_seq_hashed)
-    assert np.all(X_atom_pair_as_bit_vect == X_seq_as_bit_vect)
+def test_atom_pair_cound_fingerprint(example_molecules):
+    X = example_molecules
+    X_for_rdkit = [Chem.MolFromSmiles(x) for x in X]
+    rdkit_generator = fpgens.GetAtomPairGenerator()
+    atom_pair = AtomPairFingerprint(fingerprint_type="count", n_jobs=-1)
+    fp_function = rdkit_generator.GetCountFingerprint
+    X_emf = atom_pair.transform(X.copy())
+    X_rdkit = np.array([fp_function(x).ToList() for x in X_for_rdkit])
```

Comment:
1. I think that using `GetCountFingerprint` directly would be more readable, i.e.:
```
X_rdkit = np.array([rdkit_generator.GetCountFingerprint(x).ToList() for x in X_for_rdkit])
```
2. Maybe rename `rdkit_generator` to something shorter, e.g. `fp_gen`? In this short context it's readable

Reply:
done